src/components/Employee/Profile/hooks.ts

```diff
@@ -0,0 +1,18 @@
+import { useEmployeeAddressesCreateWorkAddressMutation } from '@gusto/embedded-api/react-query/employeeAddressesCreateWorkAddress'
+import { invalidateAllEmployeeAddressesGetWorkAddresses } from '@gusto/embedded-api/react-query/employeeAddressesGetWorkAddresses'
+import { useEmployeeAddressesUpdateWorkAddressMutation } from '@gusto/embedded-api/react-query/employeeAddressesUpdateWorkAddress'
+import { useInvalidateOnSuccess } from '@/hooks/useInvalidateOnSuccess'
+
+export const useCreateEmployeeWorkAddress = () => {
+  return useInvalidateOnSuccess({
+    invalidators: [invalidateAllEmployeeAddressesGetWorkAddresses],
```

Comment:
In these cases we only invalidate one thing per mutation, but I thought we might need to eventually support multiple invalidations for some objects, so making this take an array is a safeguard around not needing to change call sites later